bug_actiongroup_page.php

```diff
@@ -332,6 +332,14 @@
 
 			switch( $f_action ) {
 				case 'COPY':
+					print_project_option_list(
+						null,
+						false,
+						null,
+						false,
+						true
+					);
+					break;
```

Comment:
Breaking on multiple lines is not necessary, the whole function call fits in slightly more than 80 chars.

I'll fix it when I merge.